system/ee/ExpressionEngine/Addons/member/mod.member.php

```diff
@@ -2619,6 +2619,42 @@ public function custom_profile_data($typography = true)
         return ee()->TMPL->tagdata;
     }
 
+    /**
+     * Member Role Groups list
+     */
+    public function role_groups()
+    {
+        $member_id = (!ee()->TMPL->fetch_param('member_id')) ? ee()->session->userdata('member_id') : ee()->TMPL->fetch_param('member_id');
+
+        $member = ee('Model')
+            ->get('Member', $member_id)
+            ->with('RoleGroups')
+            ->all()
+            ->first();
+
+        if (!$member) {
+            return ee()->TMPL->no_results();
+        }
+
+        $vars = [];
+        foreach($member->RoleGroups->asArray() as $roleGroup) {
+            if ($roleGroup->group_id === 0 && $roleGroup->name === null) {
+                continue;
+            }
+
+            $vars[] = [
+                'role_group_id' => $roleGroup->group_id,
+                'role_group_name' => $roleGroup->name,
+            ];
+        };
+
+        if (!$vars) {
```

Comment:
Casing array to boolean is not PHP-version-proof. It should be better to use `empty()` check instead